src/std/test.ss

```diff
@@ -42,14 +45,19 @@
                check loc value expected))))
 
 (def *test-verbose* #t)
-
 (def (set-test-verbose! val)
   (set! *test-verbose* val))
+(def (test-verbose?) *test-verbose*)
 
 (def (verbose fmt . args)
   (when *test-verbose*
     (apply printf fmt args)))
 
+(def *test-quiet* #f)
+(def (set-test-quiet! val)
+  (set! *test-quiet* val))
+(def (test-quiet?) *test-quiet*)
```

Comment:
this is not used in the module... pointless.

Reply:
It is used in test cases where `invoke` could pollute stdout.